modules/admin_manual/pages/configuration/server/excluded_blacklisted_files.adoc

```diff
@@ -39,23 +50,20 @@ foreign `.htaccess` file could overwrite rules defined by ownCloud.
 There is no explicit need to enter the file name `.htaccess` as
 parameter to the `blacklisted_files` array in `config.php`, but you can
 add more blacklisted file names if necessary.
+You can also prevent uploading, e.g., Outlook .pst files, with this mechanism to avoid massive backend space allocation. 
 
 == Excluded Directories
 
-*Reason for excluding directories:*
-
-1.  Enterprise storage systems, or special filesystems like ZFS and
-BtrFS are capable of snapshots. These snapshots are directories and keep
-point-in-time views of the data.
-2.  Snapshot directories are read-only.
-3.  There is no common naming for these directories, and most likely
-will never be. NetApp uses `.snapshot` and `~snapshot`, EMC eg `.ckpt`,
-HDS eg `.latest` and `~latest`, the ZFS filesystem uses `.zfs` and so
-on.
-4.  Viewing and scanning of these directories does not make any sense as
-these directories are used to ease backup, restores, and cloning
-5.  Directories which are part of the mounted filesystem, but must not
-be accessible via ownCloud.
+=== Reasons for Excluding Directories
```

Comment:
As we now have a sub section for a reason in excluding, it would make a lot of sence to do the same for blacklisting below. This would mean to add the sub section and add eg a one liner as text example.